go/src/pluginmgr/pluginmgr_common/pluginmgr_helper.go

```diff
@@ -0,0 +1,491 @@
+package pluginmgr_common
```

Comment:
Minor: could add a one liner to describe about this file.

Reply:
added